perun/collect/bounds/meson.build

```diff
@@ -0,0 +1,22 @@
+perun_collect_bounds_dir = perun_collect_dir / 'bounds'
+
+perun_collect_bounds_files = files(
+    '__init__.py',
+    'parser.py',
+    'run.py',
+)
+
+libedit_2_dep = dependency('libedit', version: ['>=2.0.0', '<3.0.0'], required: false)
+
+#if libedit_2_dep.found()
+  py3.install_sources(
+      perun_collect_bounds_files,
+      subdir: perun_collect_bounds_dir,
+  )
+
+  install_subdir(
+      'bin',
+      install_dir: py3.get_install_dir() / perun_collect_bounds_dir,
+      install_tag: 'python-runtime',
+  )
+  #endif
```

Comment:
While going through the code I stumbled on this one remnant. How should this be handled? At least on my machine the `bound` collector will not work due to the missing dependency. Because it is not bundled with Perun, it might be worthwhile to set the dependency as a required one but a better approach would probably be to bundle it or even better update the collector to support `libedit-3`.

Proper disabling of modules can be done but it will be much more involved than just not installing it.

Reply:
This is OK for now, though we should handle the bounds collector better, you are right.